src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -229,7 +229,8 @@ public MSBuildClientExitResult Execute(CancellationToken cancellationToken)
                 CommunicationsUtilities.Trace("Build finished.");
             }
 
-            NativeMethodsShared.RestoreConsoleMode(_originalConsoleMode);
+            int STD_OUTPUT_HANDLE = -11;
```

Comment:
Can we reference the constant defined in Framework?
Or possibly make the usage of `STD_OUTPUT_HANDLE` default there